source/Calamari.AzureAppService.Tests/AppServiceBehaviourFixture.cs

```diff
@@ -423,7 +435,7 @@ public async Task CanDeployZip_ToLinuxFunctionApp()
                                         .Execute();
 
                 // Assert
-                await DoWithRetries(10,
```

Comment:
AssertContent already retries with an exponential backoff. This time has been increased, if the container isn't available in 2 runs ~8 minutes, it generally is a different failure